erpnext/docs/user/manual/en/setting-up/workflows.md

```diff
@@ -71,6 +79,23 @@ When the Leave Approver opens the Leave Application page, he can finally "Approv
 
 <img class="screenshot" alt="Workflow" src="{{docs_base_url}}/assets/img/setup/workflow-5.png">
 
+#### Workflow Actions
+
+> New in Version 11
+
+Workflow Actions facilitates communication to the users via email and also creates a list of pending Workflow Actions.
+
+For example, when an employee creates a leave application the next actions would be to `Approve` or `Reject` that application.
+Based on allowed roles to perform these actions, users will be sent email with Next Action buttons and a print format of the application.
```

Comment:
If a User is eligible to take action on some workflows, emails will be sent to him, with the relevant document as attachment, from where he can `Approve` or `Reject` the Workflow.